managed-databases/document-database/concepts.mdx

```diff
@@ -0,0 +1,127 @@
+---
+meta:
+  title: Managed Document Databases - Concepts
+  description: This page explains all the concepts related to Managed Document Databases.
+content:
+  h1: Managed Document Databases - Concepts
+  paragraph: This page explains all the concepts related to Managed Document Databases.
+tags: concepts managed-databases ferredb mongodb document
+categories:
+  - managed-databases
+---
+
+<Concept opened>
+
+## Allowed IPs
+
+Access Control List (ACL) rules [define permissions for remote access to an Instance](/managed-databases/document-database/how-to/manage-allowed-ip-addresses/). A rule consists of an IP address or an IP range. You can use them to define which host and networks can connect to your Database Instance's endpoint. You can add, edit or delete rules from your ACLs. The initial setup of a Database Instance allows full network access from anywhere (`0.0.0.0/0`).
+
+Access control is handled directly at network-level by Load Balancers, making the filtering more efficient, universal and relieving the Database Instance from this task.
+
+</Concept>
+
+<Concept opened>
+
+## Advanced settings
+
+Database Instances have settings that allow you to tune the behavior of their database engine to better fit your needs. Available settings depend on the database engine and its version. Each Database Instance setting entry has a default value that the user can override. The deletion of a setting entry restores the setting to the entry's default value. Some of the default values can be different from the engine's default, as their configuration is optimized to work at full potential with the Scaleway ecosystem.
+
+</Concept>
+
+<Concept opened>
+
+## Clone feature
+
+Allows you to create a full copy of your Database Instance (including users and permissions) at a point in time. This new Instance is fully independent of its parent.
+
+</Concept>
+
+<Concept opened>
+
+## Database snapshot
+
+A [Snapshot](/managed-databases/document-database/how-to/manage-snapshots/) is a consistent, instantaneous copy of the Block Storage volume of your Database Instance at a certain point in time. They are designed to recover your data in case of failure or accidental alterations of the data by a user. They allow you to quickly create a new Instance from a previous state of your database, regardless of the size of the volume. Snapshots can only be stored in the same location as the original data.
+
+</Concept>
+
+<Concept>
+
+## Endpoint
+
+A point of connection to a database. The endpoint is associated with an IPv4 address and a port, and contains the information of whether the endpoint is read-write or not.
+
+</Concept>
+
+<Concept>
+
+## Engine
+
+A database engine is the software component that stores and retrieves your data from a database. Currently, FerretDB 1.0.0 is available.
```

Comment:
FerretDB 1.0.0 -> FerretDB 1.2.0 (it's the version available technically, but we still need to update it in Console)
Since FerretDB also relies on PostgreSQL, we may also mention that we are currently based on  this eg. "FerretDB 1.2.0 is available, and relies itself on PostgreSQL 14.8". User doesn't have access to PostgreSQL directly and it should be transparent for them, but the information might be interesting for transparency and is already provided when connecting to the database (connection message displays "Powered by FerretDB v1.2.0 and PostgreSQL 14.8.")